core/objlist.hpp

```diff
@@ -253,6 +253,9 @@ class ObjList : public ObjListBase {
         DiskStore ds(list_name_);
         BinStream bs;
         bs << objlist_data_;
+        bs << sorted_size;
```

Comment:
Could you please help add an underline to all variables like `sorted_size_`, `del_bitmap_` and `hashed_objs_` since they're protected member?